pyartcd/pyartcd/jenkins.py

```diff
@@ -61,6 +61,14 @@ def get_build_url():
 
 
 def get_build_path():
+    """
+    Examples:
+    - https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Focp4/46870/ =>
+        job/aos-cd-builds/job/build%252Focp4/46870
+    - https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Focp4/46870 =>
```

Comment:
repeat?

Reply:
It shows that the trailing '/' is removed